components/Popup_addblockchain.js

```diff
@@ -0,0 +1,96 @@
+import styled from "styled-components";
+import Close from "../public/images/close.svg";
+import Image from "next/image";
+import {setGlobalState} from "../state";
+import {useState} from "react";
+import {StyledButton} from "./Popup_bookmarked";
+
+export default function PopupAddBlockchain({trigger, passData = () => {}}) {
+  let [options, setOptions] = useState([
+    {id: 0, blockchain: "Ethereum"},
+    {id: 1, blockchain: "Polygon"},
+    {id: 2, blockchain: "BSC"},
+    {id: 3, blockchain: "Optimism"},
+  ]);
+
+  function closePopupAdd() {
+    setGlobalState("openPopupAddBlockchain", false);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const {blockchain} = form.elements;
+    const newBlockchain = {
+      id: options.length + 1,
+      blockchain: blockchain.value,
+    };
+    setOptions(options => {
+      return [newBlockchain, ...options];
+    });
+    form.reset();
+    blockchain.focus();
+  }
+  passData(options);
+
+  return trigger ? (
+    <>
```

Comment:
You don't need a fraction `<>` element at this point. Your `<StyledPopup>` is already your top level element.

Reply:
done!